app/guid-file/styles.scss

```diff
@@ -162,6 +166,7 @@
 .edit-metadata-form {
     display: inline-flex;
     flex-direction: column;
+    max-width: 100%;
```

Comment:
This also needs fixing in the node metadata form.

Reply:
Incidentally, you should be able to reproduce this by having your developer tools open in the same window as your content and on the side rather than the bottom, then adjusting the width of the developer tools to make your main content smaller. It lets you go much smaller than just resizing the whole window.

Reply:
Using this technique also results in a large break for the main container: 

<img width="1488" alt="Pasted Graphic 77" src="https://user-images.githubusercontent.com/82047646/216368550-c8a2d1da-3524-4fc6-ae5a-34024b753e2a.png">

Reply:
When in edit this becomes much more of an issue:

<img width="1680" alt="image" src="https://user-images.githubusercontent.com/82047646/216368863-ac57f153-fa93-4f11-b6a3-d46320aaf632.png">

Reply:
> Also, in the code review template, "feature flag" refers to whether this PR relies on a Waffle flag to be active in order to function. It's not the branch that your PR is based on.

Thank you. I did not know that. Updating now.

Reply:
This should now be resolved. Please let me know if you find any other resizing or mobile issues.